pkg/portercontext/context.go

```diff
@@ -451,6 +456,16 @@ func (cw *CensoredWriter) Write(b []byte) (int, error) {
 	return len(b), err
 }
 
+func (cw *CensoredWriter) Censor(b []byte) []byte {
+	for _, val := range cw.sensitiveValues {
+		if strings.TrimSpace(val) != "" {
+			b = bytes.Replace(b, []byte(val), []byte("*******"), -1)
```

Comment:
qq: Should we replace `[]byte("*******"),` with just `[]byte("**"),`, assuming we have a very long sensitive value this can get kind of large, no?

Reply:
@schristoff  The length of the sensitive value should not matter, as it will always replace the value with `*******`. This will of course make the log longer if the sensitive value is very short, but otherwise the lenght should not matter